data/typelibrary/adapterized-1.0.0/MANIFEST.MF

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="platform:/resource/org.eclipse.fordiac.ide.library.model/model/library.xsd" Scope="Library">
+    <Dependencies>
+        <Required SymbolicName="core" Version="1.0.0"/>
+    </Dependencies>
+    <Product Name="Convert" SymbolicName="adapterized" Comment="Standard Library - Adapterized">
```

Comment:
Don't we have a more speaking name?

Reply:
Adapter-Supported
Adapter-Enabled
Adapter-Integrated
What do you think?